### ISSUE TITLE
WAI-ARIA acccesibility issue with Group

### ISSUE THREAD

**Issue by AviVahl** [bvaughn/react-resizable-panels#640]
Ran Lighthouse's audit on a page containing a `Group` with two panels.
It warned me the element representing the `Group` has an `aria-orientation`, but no matching role.

If I understand the WAI-ARIA spec correctly, I think the `Group` itself should:
- have a `role="group"`, per https://www.w3.org/TR/wai-aria/#group
- have **no** `aria-orientation`. only `Separator` should have that defined, per https://www.w3.org/TR/wai-aria/#aria-orientation

**Comment by bvaughn** [comment]
I don't think the ARIA "group" role is appropriate for this library. I briefly considered it when writing the current version but ruled it out because of the way the docs describe its use.

Quoting from the [W3 docs](https://www.w3.org/TR/wai-aria/#group):
> A set of user interface [objects](https://www.w3.org/TR/wai-aria/#dfn-object) that is not intended to be included in a page summary or table of contents by [assistive technologies](https://www.w3.org/TR/wai-aria/#dfn-assistive-technology).

Quoting from [MND docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/group_role):
> The group role should not be used for major perceivable sections of a page. If a section is significant enough that it should be included in the page's table of contents, use the region role or a standard [landmark role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles#3._landmark_roles).

I agree with you that `aria-orientation` might not belong on the `Group` element, but I worry that removing it _could_ be considered a breaking change. (People might be using it for selectors/styles.)

**Comment by bvaughn** [comment]
I guess this could also be argued to be a bugfix since the role _was_ invalid before, so I'll make the `aria-orientation` change.

**Comment by AviVahl** [comment]
Thank you. 🙏

**Comment by bvaughn** [comment]
You're welcome!

Fix just went out as part of react-resizable-panels@4.5.5

---

❤️ → ☕ [givebrian.coffee](http://givebrian.coffee/)
